src/api/index.ts

```diff
@@ -71,6 +73,18 @@ function createErc20Api(injectedDependencies: Erc20ApiDependencies): Erc20Api {
   return erc20Api
 }
 
+function createWethApi(injectedDependencies: WethApiDependencies): WethApi {
+  let wethApi
+  if (process.env.MOCK_WETH === 'true') {
+    wethApi = new WethApiMock()
+  } else {
+    wethApi = new WethApiImpl(injectedDependencies)
+  }
+  window['wethApi'] = wethApi // register for convenience
```

Comment:
I'm still unsure we even want this in production

Reply:
We do it for all the APIs, I think it's convenient. Happy to review, but should be reviewed all of our APIs (not this PR)